restclient/src/main/java/com/box/l10n/mojito/rest/entity/OpenAIPromptContextMessageCreateRequest.java

```diff
@@ -0,0 +1,44 @@
+package com.box.l10n.mojito.rest.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OpenAIPromptContextMessageCreateRequest {
```

Comment:
Just wanted to check re the naming of this class as well - as I wasn't as sure about this one